Makefile

```
@@ -55,10 +55,28 @@ og:
+	@chmod +x .git/hooks/pre-commit
+	@echo "Installed .git/hooks/pre-commit (regenerates docs/CONFORMANCE.md when src/scripts change)."
+
 test:
 	$(PYTHON) -m pytest tests/
 
 coverage:
 	$(PYTHON) -m pytest --cov=src/phonometry --cov-report=term-missing tests/
 
-check: lint security test
\ No newline at end of file
+check: lint security test
```

checkmake (0.3.2)

[warning] Target "check" should be declared PHONY.

(phonydeclared)